docs/usage/self-hosted-configuration.md

```diff
@@ -148,9 +148,17 @@ Supported tools for dynamic install are:
 - `gradle-wrapper`
 - `jb`
 - `jsonnet-bundler`
+- `lerna`
 - `mix`
+- `node`
 - `npm`
 - `pipenv`
+- `pip-compile`
+- `pip_requirements`
+- `pnpm`
+- `poetry`
+- `python`
+- `yarn`
```

Comment:
This list is not sorted from A-Z. 😉 

To fix:

1. Open the file in VSCode.
2. Select the list items.
3. Use the <kbd>F1</kbd> key to open the Command Palette.
4. Search for `Sort lines ascending`.
5. Select "Sort lines ascending" to fix the list.